README.md

```diff
@@ -0,0 +1,108 @@
+# Purple Bubble Server
+
+[![AGPL License](https://img.shields.io/badge/license-AGPL-blue.svg)](http://www.gnu.org/licenses/agpl-3.0)
+
+Purple Bubble Server is a Node.js implementation of a decentralized, secure messaging system designed to provide anonymity and security for its users.
+
+## Features
+
+- Decentralized mesh network of servers
+- RSA 4096 encryption for message security
+- Separate transmission (TX) and reception (RX) servers for enhanced anonymity
+- Message retention for 7 days
+- Automatic server and message synchronization
+
+## Prerequisites
+
+- Node.js (v14 or later recommended)
```

Comment:
We should consider bumping this to a much more current version of node, and pinning the dependency in the package file.

(See https://github.com/phish-directory/api/blob/4eb090625fb7b03251e7336310f12a255845affb/package.json#L60-L64 as an example)

Reply:
Yeah I think 18 is the newest LTS now, not sure if we wanna go for LTS or latest?